### DISCUSSION TITLE
Welcome to the Martini force field initiative discussion board!

### DISCUSSION THREAD

**Discussion by Lp0lp** [Martini-Force-Field-Initiative/MartiniDiscussions#1] [Announcements]
Welcome to the Martini force field initiative discussion board!

This discussion board aims to continue the long-standing tradition of the old [Martini forum](http://cgmartini.nl/index.php/forum), where anyone can ask Martini developers and the extended Martini community about anything concerning the Martini forcefield.

Have questions about our Lipid force fields? Head over to [our Lipids section!](https://github.com/orgs/Martini-Force-Field-Initiative/discussions/categories/lipids) Not sure why your simulation crashed? Feel free to [ask why!](https://github.com/orgs/Martini-Force-Field-Initiative/discussions/categories/q-a) 

Looking forward to discussing Martini with you all!

**Comment by minimalist25** [comment]
Dear developers

I am working with a coarse-grained MARTINI system. My system contains a beta-sheet protein, which is modeled using an elastic network, and the bilayer contains MARTINI-3 lipids. I am trying to visualize the protein without an elastic network, for which I am giving the following commands in the TK console:
vmd > source /home/user/scripts/cg_bonds.tcl
vmd > cg_bonds -top vmd.top -topoltype "elastic"

But the error that crops up is: atomsel : setbonds: Need one bondlist for each selected atom

Even if I try to visualize the elastic network protein alone using:
vmd > source /home/user/scripts/cg_secondary_structure.tcl
vmd > cg_ss -hlxcolor "green" -shtmat "AOChalky" -ssdump protein.dat
I cannot visualize the secondary structure of the protein.

I tried searching a lot on the web but did not get a clue.
How do I solve this? Any help in this regard would be really helpful.
Thanks and Regards 
Samapika

**Reply by csbrasnett** [reply]
Hi @minimalist25, you might like to try my new package I've written to help with martini visualisation in vmd: https://github.com/Martini-Force-Field-Initiative/MartiniGlass

for the two errors you have, I think the first one is probably related to not having the same system composition in your .top file as in the .gro file you're looking at. Maybe you removed water from the gro file using trjconv but not the W entry from the top?

For the second error about secondary structure, it might be that you don't have the la.tcl script ready to load for the secondary structure script to calculate how to generate the structure. I've adapted the script in the latest version so this is no longer necessary, and hopefully fixes the problem.

If you'd like to discuss this more, I'd be very grateful if you opened a new discussion in the MartiniGlass repository: https://github.com/Martini-Force-Field-Initiative/MartiniGlass/discussions so that we get discussions going there!

**Reply by minimalist25** [reply]
Thanks, Chris. I checked the gro and top. They have no issues. All the groups in gro and their order align with the top file. Still, when I proceed, I find the same error. 
Thanks for informing me about the MartiniGlass approach. I will try this out.
